python/gui/auto_generated/layout/qgslayoutdesignerinterface.sip.in

```diff
@@ -378,6 +378,13 @@ designer is in report mode.
 
 
 .. versionadded:: 3.4
+%End
+
+    virtual QgsShortcutsManager *shortcutsManager() = 0;
```

Comment:
Does this need to be exposed? I can't see how a plugin would make use of it. (You can add SIP_SKIP to the declaration line to omit it)

Reply:
Yeah, I wasn't too sure about adding it to the QgsLayoutDesignerInterface... I think I will just discard the second commit of this PR.

Reply:
I'd remove it -- if we expose them it becomes part of stable API and we lose flexibility in future.

Reply:
Done!